### DISCUSSION TITLE
What is the correct way to extract the data? (e.g. is it ok if proof_steps/human is missing)

### DISCUSSION THREAD

**Discussion by brando90** [princeton-vl/CoqGym#44] [Q&A] [answered]
I noticed that the training script has this step:
```
    parser.add_argument('--datapath', type=str, default='proof_steps/human')
```
but the `extract_proof_steps.py` create splits like this one:
```
proof_steps/train
```
which makes me wonder why the default is different from what the default options expect. I am trying to make sure I am creating the right data set when running the `proof_steps`. If there is no `human` but only `train,valid` that is what is expected right?

**Comment by yangky11** [comment, marked as answer]
It looks like the path in `extract_proof_steps.py` should also be `'proof_steps/human`. I'll update the repo after doing some tests to double check.

**Reply by yangky11** [reply]
I updated the repo to remove `/human` in ASTactic/options.py

**Comment by brando90** [comment]
Fantastic!

I plan to give a few pull requests to help out too. e.g. the python organization + I wrote a script that downloads the data from google drive automatically.

**Reply by yangky11** [reply]
thx!
